packages/styles/src/components/datepicker.scss

```diff
@@ -139,6 +146,9 @@ span.ngb-dp-navigation-chevron {
     height: datepicker.$ngb-dp-icon-size;
     transform: none;
   }
+  &:focus-visible {
+    outline: forms.$input-focus-outline-thickness solid var(--post-contrast-color);
+  }
```

Comment:
This one is not compliant with WCAG. Check this article: https://www.sarasoueidan.com/blog/focus-indicators/

You need the same style as buttons.